test/python/topology/test_functions.py

```diff
@@ -86,4 +86,22 @@ def check_int_strings_transform_with_drop(t):
       _hwcount += 1
       return None
    raise AssertionError() 
+ 
+def strings_multi_transform():
+   return ["mary had a little lamb", "its fleece was white as snow"]
+   
+global expected_strings_multi_transform
+expected_strings_multi_transform = ["mary", "had", "a", "little", "lamb", "its", "fleece", "was", "white", "as", "snow"]
 
+def split_words(t):
+   return t.split()
+ 
+def check_strings_multi_transform(t):
+   print("TUPLE", t)
+   global _hwcount
```

Comment:
Yes, I should probably use a different global variable to be safe.  I think it's with some luck that the test passes.  The test is run in standalone mode, so a new jvm is invoked, which means new python runtime, so globals are set to zero.  Also, the test case doesn't use other test functions that modify `_hwcount`.